_shared/project/tox.ini

```diff
@@ -51,6 +57,7 @@ passenv =
 deps =
 {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
     pip-tools
+    pip-sync-faster
```

Comment:
Have tox install `pip-sync-faster` as part of `tox.ini`'s `deps`. This will get tox to install the latest version of `pip-sync-faster` whenever it (re-)creates a virtualenv that didn't already exist, before it runs the `commands_pre` and `commands`. This is to fix an edge-case that I ran into: if the `requirements/*.txt` file (which contains `pip-sync-faster`) hasn't been installed into the virtualenv yet (we just have a freshly created, empty virtualenv) then when tox gets to the `pip-sync-faster requirements/*.txt` file in `tox.ini`'s `commands_pre` section it crashes because `pip-sync-faster` hasn't been installed in the virtualenv yet.

Note that there's still a pinned version of `pip-sync-faster` listed in all the `requirements/*.txt` files so `pip-sync-faster` will still upgrade (or downgrade) itself to the pinned version from the requirements file and Dependabot will still send PRs to our apps to update their `pip-sync-faster` versions. This unpinned installation of the latest version of `pip-sync-faster` directly in `tox.ini` only happens the first time a new virtualenv is created.